module/zfs/zstd/common/entropy_common.c

```diff
@@ -0,0 +1,234 @@
+/*
```

Comment:
Moving these handful of source files and private headers out of `zstd` sub-directories with make it easier to build them as `kmod`.  The Linux kernel did this as well clearly to keep the build system simple.  This shouldn't complicate pulling in new zstd versions either, `git` doesn't care about file locations.